### ISSUE TITLE
Blinking from the NPC in the hut on Yojamba Isle can put you on the roof.

### ISSUE THREAD

**Issue by madnj** [LightsHope/server#593]
I am writing this report due to the issue leading me to receive a warning for safespotting - though the alleged behavior occurred as a result of a simple blink.  Standing underneath and talking to the questgiver (the one you turn in the Hakkar head), I was stunned by a rogue and blinked out of it.  Surprisingly, the blink placed me on the roof.

This unfortunately was screenshotted and lead to a report for safespotting, however the act of getting up there was a simple blink.

The GM handling the case suggested I post it as a bug report.

**Comment by Sabre142** [comment]
Considering you can slow fall down to this hut, this shouldn't even be an offense.

**Comment by madnj** [comment]
Agreed, and reading the ToU, safespotting is the act of using terrain to provide a PVP advantage.  I blinked up there and proceeded to eat/drink - I did not retaliate or attack anyone from up there, and later the same rogue climbed up a tree and jumped to the top to kill me up there anyway, pretty much proving that the hut roof is reachable using ordinary means.  The fact that this silliness ended up with "a warning for safespotting that could lead to a ban if caught again" is frustrating when I by definition did not abuse the terrain for any advantage other than to  live for an extra 30ish seconds.
